components/husky_a300/husky_a300_replacing_fuses_board_mount.mdx

```diff
@@ -0,0 +1,19 @@
+### Replacing Board Mount Fuses  {#replacing-fuses-board-mount}
+
+1. Perform the [Lockout-Tagout Procedure](#lockout-tagout).
+1. Locate the fuse to be replaced. Follow the maintenance procedures for the corresponding
+   subassembly to access the fuse.
+   :::safety-caution
+   **Treat all circuits, cables, and fuses inside the robot as potentially live.**
+

```

Comment:
This section's ordered list is being rendered as double spaced. This blank line is likely the issue.

Reply:
Fixed